plugins/snapshots/overlay/overlayutils/fs_linux.go

```diff
@@ -0,0 +1,124 @@
+//go:build linux
+// +build linux
```

Comment:
These lines are not needed, as already implied by the filename suffix

Reply:
done